components/console-backend-service/internal/domain/k8s/versioninfo_converter_test.go

```diff
@@ -51,7 +51,7 @@ func TestVersionInfoConverter_ToGQL(t *testing.T) {
 	t.Run("Master version", func(t *testing.T) {
 		image := "eu.gcr.io/test/12345678"
```

Comment:
We should probably have the `master-12345678` since this is what the installer image actually looks like right now. Being picky, I know, sorry 😞